src/Magento/FunctionalTestingFramework/Console/RunManifestCommand.php

```diff
@@ -103,23 +104,32 @@ protected function execute(InputInterface $input, OutputInterface $output): int
      * @param string          $manifestLine
      * @param OutputInterface $output
      * @return void
+     * @throws \Exception
      *
      * @SuppressWarnings(PHPMD.UnusedLocalVariable) Need this because of the unused $type variable in the closure
      */
     private function runManifestLine(string $manifestLine, OutputInterface $output)
     {
-        $codeceptionCommand = realpath(PROJECT_ROOT . "/vendor/bin/codecept")
-            . " run functional --verbose --steps "
-            . $manifestLine;
-
-        // run the codecept command in a sub process
-        $process = new Process($codeceptionCommand);
-        $process->setWorkingDirectory(TESTS_BP);
-        $process->setIdleTimeout(600);
-        $process->setTimeout(0);
-        $subReturnCode = $process->run(function ($type, $buffer) use ($output) {
-            $output->write($buffer);
-        });
+        if (getenv('ENABLE_PAUSE') === 'true') {
+            $codeceptionCommand = BaseGenerateCommand::CODECEPT_RUN_COMMAND
```

Comment:
A space is needed after `--debug`: See error on running `run:manifest`: 

>  The "--debugtests/functional/tests/MFTF/_generated/default/TrackingScriptTestCest.php" option does not exist.

Reply:
maybe it's a good idea to add a space after --debug for all other commands too, to be safe.

Reply:
spaces are in other place